javascriptExercises/page01-81Exercises/looping-a-triangle.js

```diff
@@ -0,0 +1,5 @@
+let triangle = "#";
+for (let i = 0; i <= 7; i++) {
+    triangle = triangle + "#";
+    console.log(triangle);
+}
```

Comment:
This soluition is wrong because the first row will show two hash signs instead of one